src/components/profile/EditForm.js

```diff
@@ -1,75 +1,87 @@
-import { useState } from 'react';
-import Button from '@mui/material/Button';
-import Dialog from '@mui/material/Dialog';
-import TextField from '@mui/material/TextField';
-import DialogContent from '@mui/material/DialogContent';
-import './style.css'
+import { useState } from "react";
```

Comment:
It looks like you're changing a lot of these single quotes to double quotes. Are you able to revert them back please?

You may find that prettier is changing them automatically. In which case, you can change a setting for prettier to use single quotes instead of double.